javascript/401-code-challenges/trees/README.md

```diff
@@ -32,6 +41,10 @@ KaryTrees
 Time: Add method is O(k) worst case, contains method is O(n)
 Space: Add method is O(1), contains method is O(h)
 
+findMaxValue
+Time: O(n)
+Space: O(n)
```

Comment:
This is correct for your current code, but you should be able to do it with O(1) space.